doc/tutorials/dnn/dnn_high_level_api/dnn_classification_module.markdown

```diff
@@ -0,0 +1,61 @@
+# High Level API: Classification Module {#tutorial_dnn_classification_module}
+
+### Introduction
+
+In this tutorial we will present how easy it is to run inference on a classification model using the new OpenCV Dnn API.
+
+### Usage
+
+Here is a simple snippet of the code in C++ with which you can run inference on a classification model.
+In this case the model is a simple AlexNet CNN  trained on ImageNet.
+
+~~~~~~~~~~~~~{.cpp}
+
+#include <opencv2/core.hpp>
+#include <opencv2/highgui/highgui.hpp>
```

Comment:
Please extract code from text documents and embed it through `@include` or `@snippet` doxygen commands.

Dedicated locations for tutorial files (or reuse any existed sample from "samples/" folder):
- https://github.com/opencv/opencv/tree/master/samples/cpp/tutorial_code
- https://github.com/opencv/opencv/tree/master/samples/python/tutorial_code

This help to ensure that code in documentation is not broken:
- All C++ samples are checked through compilation on CI.
- Python files are checked by linters (pylint / flake8 - see "Docs" builder).

Reply:
Hi, @alalek, I have a couple of questions:

1. Will the CI be able to  check the samples if they have runtime parameters such as the ones in detection.cpp or classification.cpp?

2. The code that I have inserted in the markdown is simpler than the one in the samples, since I wanted to make the example easier to understand, can I include a block of code with @code ... @endcode instead of embedding a file? If not, where should I create a new file and put this block of code in order to embed it using @snippet?

3.  Some of the required files for the samples to run are network weights, these files are too large to make them part of the tutorial files and upload them to the repo, what should I do?

Reply:
1. CI just checks compilation (in case of C++). Samples are not launched.

2. Normal "samples" contains a lot of helper code, like args parsing, etc. Code in tutorials is more simple. There are examples of [code block](https://github.com/opencv/opencv/blame/8b3042da63f4d0f112a1ec2199ebffc2894f567d/samples/cpp/tutorial_code/core/file_input_output/file_input_output.cpp#L156-L160) and [its embedding](https://github.com/opencv/opencv/blame/8b3042da63f4d0f112a1ec2199ebffc2894f567d/doc/tutorials/core/file_input_output_with_xml_yml/file_input_output_with_xml_yml.markdown#L137-L142)

3. It is better to reuse already existed data in repository. If not, then external link (URL) is preferable in case of huge data. Users should be able to reproduce tutorial results.

Reply:
3. The external data that I use its already part of the downloadable links in [opencv_extra](https://github.com/opencv/opencv_extra/blob/master/testdata/dnn/download_models.py), should I just explain this in the tutorial, or just comment the URL in the tutorial file so that the users can download it?

Reply:
I believe, it is enough to add reference with model's name and with the link on downloader script.